common/djangoapps/microsite_configuration/models.py

```diff
@@ -0,0 +1,181 @@
+"""
```

Comment:
This seems like a good opportunity to establish a relationship with the Django "Site" model -- what do you think about adding a foreign key to the tables to link back to a particular site?  Would this be a helpful step toward eventual combination with comprehensive theming?  Also, if we are going to link to Site here, we should consider how it might be accomplished with the settings file as well.